airbyte-integrations/connectors/source-omnisend/source_omnisend/omnisend.yaml

```diff
@@ -0,0 +1,83 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["{{ options['name'] }}"]
+  requester:
+    url_base: "https://api.omnisend.com/v3"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: X-API-KEY
+      api_token: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      url_base: "*ref(definitions.requester.url_base)"
+      page_size_option:
+        inject_into: "request_parameter"
+        field_name: "limit"
+      pagination_strategy:
+        type: CursorPagination
+        cursor_value: "{{response['paging']['next']}}"
```

Comment:
Do you not need the stop condition?

Reply:
When there are no next pages, `response['paging']['next']` is `null`, so not sure if it is necessary to explicitly add the condition? I added it nonetheless, will remove if it is not necessary.